Templates/TypeScript/Operation.stencil

```diff
@@ -25,7 +25,7 @@ const document: SyrupOperation<{{ name }}, {% if operation.variables.count > 0 %
   id: "{{ queryString|encryptData }}",
   name: "{{ operation.name }}",
   source: "{{ queryString|replace:"$","\$"|replaceTypeScriptQuotes }}",
-  operationType: '{{ operation|renderOperationTypeName|lowercase } }}',
```

Comment:
I think this was a bug introduced a few years ago. Not sure how tests were passing before 🤔

Reply:
This fails local and server CI after updating stencil. My best guess is that it used to ignore the extra character at the end and now is catching the mis matched brace. Everything seems to look good after removing it.